lib/api/createRpc.ts

```diff
@@ -5,7 +5,7 @@ import { subscriptionMethods } from '@lightninglabs/lnc-core';
 const capitalize = (s: string) => s && s[0].toUpperCase() + s.slice(1);
 
 /**
- * Creates a typed Proxy object which calls the WASM request or
+ * Creates a typed Proxy object which calls the mobile request or
```

Comment:
seems like the word "mobile" is too vague here. what does it add? what about just deleting the word "mobile" completely?